californium-core/src/test/java/org/eclipse/californium/core/test/CountingCoapHandler.java

```diff
@@ -74,7 +74,7 @@ public final void onLoad(CoapResponse response) {
 			}
 			notifyAll();
 		}
-		LOGGER.info("Received {}. Notification: {}", counter, response.advanced());
+		LOGGER.trace("Received {}. Notification: {}", counter, response.advanced());
```

Comment:
That's a unit test. The logging is intended to analyze the effect of changes somewhere else. So I would keep info.